torch/csrc/distributed/rpc/init.cpp

```diff
@@ -342,7 +325,12 @@ If the future completes with an error, an exception is thrown.
       .def(
           "shutdown",
           &ProcessGroupAgent::shutdown,
-          py::call_guard<py::gil_scoped_release>())
+          py::call_guard<py::gil_scoped_release>(),
+          R"(
+          future.wait() should not be called after shutdown(), e.g.,
```

Comment:
Looks like this is general doc info for all RpcAgent impls? In that case, shall we add it the `shutdown` API doc in `rpc/api.py`?

Reply:
BTW, you mean non-graceful shutdown, right? Because graceful shutdown would wait for those futures?

Reply:
@mrshenli ThriftAgent does not wait for pending futures. So it's still recommended that users don't call `future.wait()` after graceful shutdown.